xml/System.Net.Http/MultipartContent.xml

```diff
@@ -212,10 +212,19 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="0" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="cancellationToken">The cancellation token to cancel the operation.</param>
+        <summary>Serializes the HTTP content to a stream using the multipart/* encoding.</summary>
```

Comment:
What does `multipart/*` mean?

Reply:
the `*` is `subtype` from constructor. `multipart/*` is used across this class doc